### PR TITLE
Accepts form format in example model.

### PR DESCRIPTION
**Pull request by shimastripe**
## WHY

Accepts `form` format request.

#65 

## WHAT

- Add `form:fieldname` tags to an existing model.
- Add the discription of `data type` in `readme.md`.

### CODE REVIEW

**Comment by shimastripe** [comment]
Sorry... I found a bug.
Please wait for a while.

**Comment by shimastripe** [comment]
@dtan4 The bug is not related to this PR.
So later, I make another PR for the bug.

File: README.md [hunk 1]
```diff
@@ -206,6 +206,38 @@ Resource name is written in the plural form.
 |`order=`|Order of items|`desc`|`asc`|
 |`v=`|API version|(empty)|`1.2.0`|
 
+### Data Type
```
Comment (dtan4) [inline README.md]: Please insert the linkg this section into table of contents.

File: README.md [hunk 2]
```diff
@@ -206,6 +206,38 @@ Resource name is written in the plural form.
...
+`multipart/form-data`
+
+```
+curl -X POST http://localhost:8080/users \
+     -F 'field=value'
+```
+
+#### Response
+
+Return `application/json` response.
```
Comment (dtan4) [inline README.md]: > Response data type is always `application/json`.

**Comment by shimastripe** [comment]
@dtan4 Thanks! I fixed.

**Comment by dtan4** [comment]
LGTM 👍

**Pull request merged**